crates/qt-build-utils/Cargo.toml

```diff
@@ -17,6 +17,7 @@ rust-version.workspace = true
 cc.workspace = true
 versions = "6.3"
 thiserror.workspace = true
+cmake-package = { path = "/home/josh/sources/cmake-package-rs", optional = true }
```

Comment:
Temporary hack, waiting for https://github.com/danvratil/cmake-package-rs/pull/48 to land upstream.

Reply:
Fixed, 0.1.5 is released with my changes